### PR TITLE
[fe](default parameters) change remote_fragment_exec_timeout_ms from 5s to 30s

### PR DESCRIPTION
**Pull request by yiguolei**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by wm1581066 — APPROVED** [review]
LGTM

**Comment by yiguolei** [comment]
run  buildall

File: fe/fe-common/src/main/java/org/apache/doris/common/Config.java [hunk 1]
```diff
@@ -1627,7 +1627,7 @@ public class Config extends ConfigBase {
      * condition，try to set this timeout longer.
      */
     @ConfField(mutable = true)
-    public static long remote_fragment_exec_timeout_ms = 5000; // 5 sec
+    public static long remote_fragment_exec_timeout_ms = 30000; // 30s sec
```
Comment (cambyzju) [inline fe/fe-common/src/main/java/org/apache/doris/common/Config.java:1630]: `30s` or `30 sec`

**Review by HappenLee — APPROVED** [review]
LGTM

**Pull request merged**